velox/docs/functions/presto/datetime.rst

```diff
@@ -102,6 +102,10 @@ Date and Time Functions
     Returns the UNIX timestamp ``unixtime`` as a timestamp with time zone
     using ``string`` for the time zone.
 
+.. function:: to_iso8601(x) -> varchar
+
+    Formats date ``x`` as an ISO 8601 string.
```

Comment:
Elsewhere, we document supported types as "Supported types for x are: A, B, C." Consider using the same style here.

```
Formats ``x`` as an ISO 8601 string. Supported types for x are: DATE.
```

Reply:
Rewrote.